### ISSUE TITLE
Prefix training job with username and timestamp for lab

### ISSUE THREAD

**Issue by w601sxs** [awslabs/amazon-sagemaker-examples#753]
Customers running this in groups (for immersion days) would like to add a username field to the base job name

**Comment by ishaaq** [comment]
Have you got a notebook that you're following? Why can't you add code to it to include the desired prefix?

**Comment by w601sxs** [comment]
Yes, easy to add code, and that's what we do now. But when running workshops for customers in large groups, we have to instruct them to modify these notebooks, while they are wondering why they need to do this for a canned lab. Just for convenience, nothing else.

**Comment by ishaaq** [comment]
Well, there are a lot of notebooks in this repo written by a large number of individuals/teams, and the primary audience is not necessarily only those in workshops so it's unlikely that what you're asking for is practically possible, but, if there is a specific notebook you are interested in then I can try and route this question to the individual/team responsible for that notebook and let them answer more definitively.

**Comment by w601sxs** [comment]
No changes required. Thanks for looking into it.
